tests/dibi/PdoDriver.providedConnection.phpt

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @dataProvider ../databases.ini
+ */
+
+declare(strict_types=1);
+
+use Tester\Assert;
+
+require __DIR__ . '/bootstrap.php';
+
+if($config['driver'] !== 'pdo') {
+	\Tester\Environment::skip('Only relevant for PDO driver.');
```

Comment:
Wanted to use something like:
```php
 /**
  * @dataProvider ../databases.ini *, pdo
  */
```
However it looks like tester [does not support](https://github.com/nette/tester/blob/f78a4bfedb78de6bdfc82fbfac064e8b6a59a5a2/src/Framework/DataProvider.php#L62) anything like that.

Reply:
Maybe use workaround like `!= foo, pdo`

Reply:
Sounds good!